lib/lockstep_rails/version.rb

```diff
@@ -1,3 +1,3 @@
 module LockstepRails
-  VERSION = "0.3.39"
```

Comment:
i see even sumit MR is open just make sure who ever goes first the second person does update the version again

Reply:
Yea that makes sense :)

Reply:
this is on the reviewer though who merges which PR first to inform the Dev 😅